ultralytics/utils/plotting.py

```diff
@@ -582,12 +582,21 @@ def plot_labels(boxes, cls, names=(), save_dir=Path(""), on_plot=None):
     LOGGER.info(f"Plotting labels to {save_dir / 'labels.jpg'}... ")
     nc = int(cls.max() + 1)  # number of classes
     boxes = boxes[:1000000]  # limit to 1M boxes
-    x = pandas.DataFrame(boxes, columns=["x", "y", "width", "height"])
+    x = polars.DataFrame(boxes, schema=["x", "y", "width", "height"])
 
     try:  # Seaborn correlogram
+        import pandas as pd
         import seaborn
 
```

Comment:
Let's eliminate the correlogram and we can get two benefits in one by eliminating both seaborn and pandas right?

We'll be displaying many of these plots interactively in Platform anyway :) @ultralytics/platform

Reply:
Done and removed.